packages/shared/lib/units.ts

```diff
@@ -2,6 +2,13 @@ import { Unit } from '@iota/unit-converter'
 import Big from 'big.js'
 import { getCurrencyPosition, formatNumber } from 'shared/lib/currency'
 
+export const MILLISECONDS_PER_SECOND = 1000
+export const SECONDS_PER_MINUTE = 60
+export const MINUTES_PER_HOUR = 60
+export const HOURS_PER_DAY = 24
+export const DAYS_PER_WEEK = 7
+export const MONTHS_PER_YEAR = 12
```

Comment:
```
export const MILLISECONDS_PER_SECOND = 1000
export const SECONDS_PER_MINUTE = 60
export const MINUTES_PER_HOUR = 60
export const HOURS_PER_DAY = 24
export const DAYS_PER_WEEK = 7
export const MONTHS_PER_YEAR = 12
```
Not sure this is the best file to store these constants, this file seems to be responsible for the units of iota. Did you check out the time-constants package?

Reply:
Or we make our own file for these constants to reduce dependencies.